libs/sdk-ui-kit/src/Header/typings.ts

```diff
@@ -49,6 +49,9 @@ export interface IAppHeaderProps {
 
     helpRedirectUrl?: string;
     theme?: ITheme;
+
+    shouldDisplayUpsellButton?: boolean;
```

Comment:
What about showUpsellButton (it's shorter)

Reply:
That's great. I changed it thanks

Reply:
Thank you :)